components/characterProfile/characterProfile.module.css

```diff
@@ -0,0 +1,12 @@
+.profileImage {
+    margin-top: 20px;
```

Comment:
FYI, Chakra UI does let you add most styles directly to the element via props (e.g. `marginTop` etc) - but if we wanna write custom CSS this way, that's okay with me.